### PR TITLE
UI Changes for barchart

### PR DESCRIPTION
**Pull request by vipulgoyal006**
Targets #{1055}

### Description
Changed the background of the tooltip to dark grey and updated the color of the text to white. It is easily readable now.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/js/app.js [hunk 1]
```diff
@@ -1035,8 +1035,8 @@ function barChart(){
 		  displayColors:false,
 		  bodyFontSize: 20,
 		  bodyFontStyle: "bold",
-		  backgroundColor:"#E5E5E5",
-		  bodyFontColor:"#E0DEDE",
+		  backgroundColor:"#282828",
+		  bodyFontColor:"#ffffff",
```
Comment (GauravGusain98) [inline resources/js/app.js:1039]: Just wanted to know, if these color codings are some randomly chosen color or you found these color from chart js website?

**Review by GauravGusain98 — COMMENTED** [review]
Looks good. Left some comments, please check.

@kuldeep3  Could you also please check this PR once?

File: resources/js/app.js (hunk 1 above)
Comment (GauravGusain98) [reply resources/js/app.js:1039]: If possible can you also share the screenshot of the new chart?

**Comment by GauravGusain98** [comment]
@vipulgoyal006 I reviewed this as it was looking ready for review to me. Just a note, If a PR is ready, please add a label for review as well.

File: resources/js/app.js (hunk 1 above)
Comment (vipulgoyal006) [reply resources/js/app.js:1039]: @GauravGusain98  I have shared the screenshots on the issue page. Please refer to the comments on issue #1055. 

For the color coding, it was mentioned in the description. So I have implemented the same.

Comment (GauravGusain98) [reply resources/js/app.js:1039]: Cool. Then its fine

**Review by GauravGusain98 — APPROVED** [review]
Looks good

**Pull request merged**